.goreleaser.yml

```diff
@@ -0,0 +1,25 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+archives:
+  - 
+    name_template: "{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}"
```

Comment:
remove new line to match the rest of the file

Reply:
Done.